testdata/config/prometheus.yaml

```diff
@@ -41,7 +41,7 @@ metadata:
   name: prommetrics
   namespace: istio-config-default
 spec:
-  selector: "true"
+  #match: request.method == "/login"
```

Comment:
why add the commented out `request.method` bit here?  I almost feel that is adding more confusion than just dropping `match` altogether`.

Reply:
dropped.